.gitignore

```diff
@@ -1,32 +1,5 @@
 # Godot auto generated files
-core/global_defaults.cpp
```

Comment:
Though this is correct, it might be better to keep the old ignores for a while to avoid having half of our contributors committing obsolete files over the next 2 months :)